src/UI/Component/Table/Factory.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/* Copyright (c) 2017 Nils Haagen <nhaageng@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Table;
+
+/**
+ * Table factory
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Presentation Table lists some data from the system in a pleasant way.
+	 *       The user should be able to get a quick overview over records in a
```

Comment:
Somehow also add the "easy to explore" aspect.

Reply:
Tried to. Better now?